src/controlCenter/widgets/menubar/menubar.vala

```diff
@@ -92,14 +94,19 @@ namespace SwayNotificationCenter.Widgets {
 
                     Gtk.Box menu = new Gtk.Box (Gtk.Orientation.VERTICAL, 0);
                     menu.get_style_context ().add_class (obj.name);
-                    obj.menu = menu;
+
+                    Gtk.Revealer r = new Gtk.Revealer ();
+                    r.add (menu);
+                    r.set_transition_duration (obj.animation_duration);
+                    r.set_transition_type (obj.animation_type);
+                    obj.revealer = r;
 
                     show_button.clicked.connect (() => {
-                    bool visible = !menu.visible;
+                    bool visible = !r.get_reveal_child ();
                     foreach (var o in menu_objects) {
-                        o.menu ?.set_visible (false);
+                        o.revealer ?.set_reveal_child (false);
                     }
-                    menu.set_visible (visible);
+                    r.set_reveal_child (visible);
                 });
```

Comment:
Indent this. This is a really annoying Uncrustify issue that I've not been able to solve yet...